packages/cli/cli.ts

```diff
@@ -53,22 +56,47 @@ if (!args['--message']) {
   throw new Error('--message is required');
 }
 
-const commently = new Commently({
+interface CommentArguments {
+  pr?: number;
+  owner?: string;
+  repo?: string;
+  title?: string;
+  key?: string;
+  useHistory?: boolean,
+  commit?: string;
+  message: string;
+}
+
+interface CommentResponse {
+  data?: {
+    html_url?: string; // eslint-disable-line camelcase
+  }
+}
+
+const addComment = (commentArgs: CommentArguments) => {
+  const CorrectCommently = commentArgs.commit ? Gitlab : Github;
+  const commently = new CorrectCommently(commentArgs)
+  commently
+    .autoComment(commentArgs.message)
+    .then((response: CommentResponse) => {
+      console.log(symbols.success, `Successfully commented on ${commentArgs.pr || commentArgs.commit}`);
```

Comment:
Thoughts on using `debug` here so we can put some of the logging behind an environment variable 

https://www.npmjs.com/package/debug